.clang-format

```diff
@@ -9,4 +9,5 @@ BinPackArguments: false
 AlignAfterOpenBracket: AlwaysBreak
 ContinuationIndentWidth: 2
 ConstructorInitializerIndentWidth: 2
+SortIncludes: false
```

Comment:
This is intended that the includes are sorted by `clang-format`.

Reply:
Hmm I think it sorted the includes in a file where I didn't add any new includes and I didn't want to mess up the project style. I'll revert this change.